backstop_data/engine_scripts/puppet/page-objects/crm-page.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+module.exports = class CrmPage {
+
+  constructor(engine, scenario, viewPort) {
+    this.engine = engine;
+    this.scenario = scenario;
+    this.viewPort = viewPort;
+  }
+
+  /**
+   * Waits and clicks every element that matches the target selector.
+   *
+   * @param {String} selector - the css selector of the target elements to
+   * click.
+   */
+  async clickAll (selector) {
+    await this.waitForSelectorAndEvaluate(selector, selector => {
+      document.querySelectorAll(selector).forEach(element => element.click());
+    });
+  }
+
+  /**
+   * Clicks on a specific option inside the currently active select2 dropdown.
+   *
+   * @param {Number} nth - the number of the option to click on. This is a zero
+   * based index.
+   */
+  async clickSelect2NthOption(nth) {
```

Comment:
I thought we were going to select by label, better not leave something we are not going to use.

Reply:
in the next ticket sorry, this one already took a huge amount of time